packages/web/localizations/__tests__/localizations.test.ts

```diff
@@ -9,6 +9,9 @@ import { glob } from "glob";
 // use to silence warnings
 const warn = (..._args: Parameters<typeof console.warn>) => null; // console.warn(...args);
 
+/** Add key paths here to skip them in localizations tests. */
+const omittedKeyPaths = ["assets.categories"];
```

Comment:
Can you give me more context on why this key is being omitted?

Reply:
I dynamically append the generated category ID to `assets.categories` translation key, so I do `"assets.categories" + category` in the code